CSLabs.Api/Controllers/SystemMessageController.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using CSLabs.Api.Models;
+
+namespace CSLabs.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class SystemMessageController : BaseController
+    {
+
+        public SystemMessageController(BaseControllerDependencies deps) : base(deps)
+        {
+        }
+
+        // GET: api/SystemMessage
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<SystemMessage>>> GetSystemMessages()
+        {
+            if (GetUser() == null)
```

Comment:
There's no need for this, just preface this method with an `[Authorize]` imperative, and it will require authorization to use. 

For your reference: https://docs.microsoft.com/en-us/aspnet/core/security/authorization/simple?view=aspnetcore-5.0

Reply:
Thanks, It still has the same issue of needing the refresh to go into effect.

Reply:
My mistake I read that incorrectly.